src/constants/errors.ts

```diff
@@ -0,0 +1,6 @@
+import { getBytesAsString } from "utils/string";
+import { LOG_LINE_SIZE_LIMIT } from "./logs";
+
+export const LOG_LINE_TOO_LARGE_ERROR = `Parsley was unable to process one or more lines due to performance reasons since they exceed the line size limit of ${getBytesAsString(
```

Comment:
[nit] May be useful to reflect the severity of the message
```suggestion
export const LOG_LINE_TOO_LARGE_WARNING = `Parsley was unable to process one or more lines due to performance reasons since they exceed the line size limit of ${getBytesAsString(
```